docs/internal-documentation/maintenance/ReleaseProcess-create-docker-image.md

```diff
@@ -32,12 +32,12 @@ same directory. If you run `cd`, then you have to run `cd` back to the start
 directory.
 
 The Docker environment is assumed to be clean. Consider running the following
-commands before proceeding (see section "[Handy Docker commands]"):
+commands before proceeding (see section "[Handy Docker commands]") to clean up:
```

Comment:
Perhaps `Consider running the following commands before proceeding [...] to clean up:` -> `Consider running the following commands to clean up before proceeding [...]:`?

Reply:
OK, that is also fine.